its/ruling/src/test/java/org/sonar/python/it/PythonExtendedRulingTest.java

```diff
@@ -38,15 +38,18 @@ class PythonExtendedRulingTest {
   @RegisterExtension
   public static final OrchestratorExtension ORCHESTRATOR = getOrchestrator();
 
-  private static final String PROFILE_NAME = "customProfile";
+  private static final String PROFILE_NAME = "rules";
 
   @BeforeAll
   static void prepare_quality_profile() throws IOException {
-    List<String> ruleKeys = bugRuleKeys();
```

Comment:
We can now remove this method (and close the associated ticket, after an honorable 3 years of waiting patiently to be handled).